### PR TITLE
Mtc controller config messages

### PR DESCRIPTION
**Pull request by TomLonergan03**
Possible issues with config messages

### CODE REVIEW

File: src/propulsion/messages.hpp [hunk 1]
```diff
@@ -22,9 +22,9 @@ static constexpr std::array<ControllerMessage, 24> kConfigurationMessages = {{
   // Set feedback type Hall sensor
   {0x2F, 0x40, 0x20, 0x01, 0x01, 0x00, 0x00, 0x00},
   // Set feedback direction to inverted
-  {0x2B, 0x40, 0x20, 0x03, 0x01, 0x00, 0x00, 0x00},
+  {0x2B, 0x40, 0x20, 0x03, 0x01, 0x00, 0x00, 0x00},//n not in manual
```
Comment (TomLonergan03) [inline src/propulsion/messages.hpp:25]: this one

File: src/propulsion/messages.hpp [hunk 2]
```diff
@@ -44,11 +44,11 @@ static constexpr std::array<ControllerMessage, 24> kConfigurationMessages = {{
   // Set current control torque regulator P gain to 16000
   {0x2B, 0xF6, 0x60, 0x01, 0x80, 0x3E, 0x00, 0x00},
   // Set current control torque regulator I gain to 1800
-  {0x2B, 0xF6, 0x60, 0x02, 0x88, 0x13, 0x00, 0x00},
+  {0x2B, 0xF6, 0x60, 0x02, 0x88, 0x13, 0x00, 0x00},//n sets to 5000
```
Comment (TomLonergan03) [inline src/propulsion/messages.hpp:47]: this one

File: src/propulsion/messages.hpp [hunk 3]
```diff
@@ -44,11 +44,11 @@ static constexpr std::array<ControllerMessage, 24> kConfigurationMessages = {{
...
   // Set current control flux regulator P gain to 2600
-  {0x2B, 0xF6, 0x60, 0x03, 0x80, 0x3E, 0x00, 0x00},
+  {0x2B, 0xF6, 0x60, 0x03, 0x80, 0x3E, 0x00, 0x00},//n sets to 16000
```
Comment (TomLonergan03) [inline src/propulsion/messages.hpp:49]: this one

File: src/propulsion/messages.hpp [hunk 4]
```diff
@@ -44,11 +44,11 @@ static constexpr std::array<ControllerMessage, 24> kConfigurationMessages = {{
...
   // Set current control flux regulator I gain to 1800
-  {0x2B, 0xF6, 0x60, 0x04, 0x88, 0x13, 0x00, 0x00},
+  {0x2B, 0xF6, 0x60, 0x04, 0x88, 0x13, 0x00, 0x00},//n sets to 5000
```
Comment (TomLonergan03) [inline src/propulsion/messages.hpp:51]: and this one

**Review by MiltFra — CHANGES_REQUESTED** [review]
Alright, I agree with your observations. 
Please a) write down *all* the variables we need to check and b) talk to propulsion about the intended values. I'm assuming we'll add some human-readable configs for that, but that's yet to be determined.

**Comment by MiltFra** [comment]
Also, this is just a copy of the cleanup PR, right?

**Pull request closed without merging**